### PR TITLE
[14.0][ADD] account_move_line_check_number

### PR DESCRIPTION
**Pull request by ps-tubtim**
This module adds the check number in the journal items.

**Example**
Selection_145

**Note:** You can use this module and `bank_statement_check_number` (https://github.com/OCA/account-reconcile/pull/402) to help to reconcile with the bank statements by check number.

### CODE REVIEW

**Review by Saran440 — APPROVED** [review]
LGTM :+1:

File: account_move_line_check_number/models/account_move.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        store=True,
+        readonly=False,
+        copy=False,
+        index=True,
+    )
+
+    @api.depends("payment_id", "payment_id.check_number")
+    def _compute_check_number(self):
+        for rec in self:
+            rec.check_number = rec.payment_id and rec.payment_id.check_number or ""
```
Comment (ChrisOForgeFlow) [inline account_move_line_check_number/models/account_move.py]: Is more efficient read direct field, if empty should return empty string
```suggestion
            rec.check_number = rec.payment_id.check_number or ""
```

**Review by ChrisOForgeFlow — CHANGES_REQUESTED** [review]

**Comment by Saran440** [comment]
@ChrisOForgeFlow Thank you for your advise, please review and approve (if it okay)

**Review by ChrisOForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**